src/App.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Title = styled.h1`
+  color: blue;
+`;
```

Comment:
Why you're using a different line?
is it just a cosmetic decision or will it break the code if you will use:
```
color: blue;`;
```

Reply:
it's just looks awful like this 
```
color: blue;`;
```